s3.tf

```diff
@@ -144,7 +144,7 @@ data "aws_iam_policy_document" "bucket" {
 
     principals {
       type        = "AWS"
-      identifiers = [for acct in data.aws_redshift_service_account.this : acct.arn]
+      identifiers = [data.aws_redshift_service_account.this.arn]
```

Comment:
this failed to apply

Reply:
so i changed it

Reply:
Since that resource doesn't use `for_each` that (failure) would be expected, so this definitely broke at some point.